web/src/styles/lifecycle.css

```
@@ -298,7 +1988,7 @@ html[data-theme="noir-gold"] {
   width: 6px;
   height: 6px;
   border-radius: 50%;
-  background: currentcolor;
+  background: currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)